bootstrapvz/common/tasks/grub.py

```diff
@@ -78,7 +78,7 @@ def run(cls, info):
 class InstallGrub_2(Task):
 	description = 'Installing grub 2'
 	phase = phases.system_modification
-	predecessors = [filesystem.FStab]
+	predecessors = [ConfigureGrub]
```

Comment:
Wow, that's a serious oversight. `update-grub` and `grub-install` depend heavily on what's in the defaults. This should be done for 1.99 as well.